src/components/taginput/Taginput.vue

```diff
@@ -18,7 +18,12 @@
                 :closable="closable"
                 :title="ellipsis && getNormalizedTagText(tag)"
                 @close="removeTag(index)">
-                {{ getNormalizedTagText(tag) }}
+                <slot
+                    name="tag"
+                    :text="getNormalizedTagText(tag)"
```

Comment:
In this case I think we might remove it because there is already tag parameter

Reply:
I've removed it in the latest commit.